x/claims/migrations/v2/migration.go

```diff
@@ -19,3 +20,26 @@ func MigrateStore(ctx sdk.Context, paramstore *paramtypes.Subspace) error {
 	paramstore.Set(ctx, types.ParamStoreKeyEVMChannels, types.DefaultEVMChannels)
 	return nil
 }
+
+// MigrateJSON accepts exported 1 x/claims genesis state and migrates it
+// to 2 x/claims genesis state. The migration includes:
+// - Add AuthorizedChannels and EVMChannels
+func MigrateJSON(oldState v1types.GenesisState) types.GenesisState {
+	finalClaims := []types.ClaimsRecordAddress{}
+	for _, claim := range oldState.ClaimsRecords {
+		finalClaims = append(finalClaims, types.ClaimsRecordAddress(claim))
+	}
```

Comment:
is this copying the claims? If so you can use the following: 

https://github.com/uber-go/guide/blob/master/style.md#copy-slices-and-maps-at-boundaries